### PR TITLE
Remove health apps

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This PR removes the health tools apps (rx, messaging, health-records, and health-beta), because they're unused and removing them now makes updating to React 16 easier.

Some of the code in messaging and rx was used in the dashboard, so I copied it into the dashboard folder and stripped out the unused code. There are likely still some unused styles, but this PR is already too big.

In order to review this, you can look through the changes in the dashboard app, starting here: https://github.com/department-of-veterans-affairs/vets-website/pull/9331/files#diff-85d04b107615dd64329299a1cc8f9a5e

## Testing done

Automated tests, checked that dashboard still opens

## Acceptance criteria
- [x] Dashboard works and still populates health info

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -232,7 +229,6 @@
     "react-addons-perf": "^15.3.0",
     "react-autosuggest-ie11-compatible": "^8.0.6",
     "react-cropper": "^1.0.1",
-    "react-datepicker": "^0.29.0",
```
Comment (ncksllvn) [inline package.json]: Was this the only module unique to these apps? Just wondering if there could be any more more dependencies that we could remove.

File: src/applications/personalization/dashboard/reducers/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import claimsAppeals from '../../../claims-status/reducers';
-import prescriptions from '../../../rx/reducers/prescriptions';
-import recipients from '../../../messaging/reducers/recipients';
-import folders from '../../../messaging/reducers/folders';
+import prescriptions from './prescriptions';
+import recipients from './recipients';
+import folders from './folders';
```
Comment (ncksllvn) [inline src/applications/personalization/dashboard/reducers/index.js]: 👍 👍

**Review by ncksllvn — APPROVED** [review]
LGTM!

File: src/applications/personalization/dashboard/utils/helpers.js [hunk 3]
```diff
@@ -1,9 +1,21 @@
 import React from 'react';
-import _ from 'lodash';
 import moment from 'moment';
 
-import environment from '../../../platform/utilities/environment';
-import { apiRequest as commonApiClient } from '../../../platform/utilities/api';
+import environment from '../../../../platform/utilities/environment';
+import { apiRequest as commonApiClient } from '../../../../platform/utilities/api';
+
+export function formatDate(date, options = {}) {
```
Comment (U-DON) [inline src/applications/personalization/dashboard/utils/helpers.js]: There's also a `formattedDate` in this same module. I'll leave it to @rianfowler to decide whether this `formatDate` be removed in favor of the former when it comes to usage in dashboard.

**Review by U-DON — APPROVED** [review]
LGTM.

File: package.json (hunk 1 above)
Comment (rianfowler) [reply package.json]: I tried a quick run of `depcheck` but it gave me a bunch of false positives. I think there are a couple of other utilities we could try.

**Comment by jbalboni** [comment]
@U-DON @ncksllvn @bshyong Do any of you know how to test the rx and messaging functionality on the dashboard? I haven't found a user that works locally.

**Review by rianfowler — APPROVED** [review]
runs for me locally

**Comment by ncksllvn** [comment]
@jbalboni Our go-to user has been 264, but I think you need to remove a [filter](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/dashboard/containers/PrescriptionsWidget.jsx#L127) applied to the health data based on the date. I think the Betamocks timestamp needs to be made dynamic for that to work otherwise.

**Comment by bshyong** [comment]
^ What Nick said. Dashboard only shows recent prescriptions and messages. The 264 user is the one to use

**Review by bshyong — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (jbalboni) [reply package.json]: This was the only one I found when updating React deps for another PR.

**Pull request merged**